config/cli.go

```diff
@@ -283,3 +371,47 @@ func validateHost(host string) error {
 
 	return nil
 }
+
+func ParseDependencySet(dependencySet string) ([]uint64, error) {
```

Comment:
does this have to be public or can it be kept internal to the cli? also if you want to use this for a unit test, I believe that you can still call it as long as the unit test has its package set to `package config`. Sorry I didnt catch this the first time around